PowerPointLabs/PowerPointLabs/TextCollection/ColorsLabText.cs

```diff
@@ -2,13 +2,16 @@
 {
     internal static class ColorsLabText
     {
-        public const string PaneTag = "ColorsLab";
-
-        public const string TaskPanelTitle = "Colors Lab";
+        #region Action Framework Tags
+        public const string RibbonMenuId = "ColorsLabButton";
```

Comment:
It has to be the same name as the id strings in the XML file (Ribbon1.xml) and for these labs, the menu buttons are called buttons. I'm not sure if I should change how these labs work because they are legacy labs that are quite old.